config/routes.rb

```diff
@@ -41,25 +42,19 @@ def add_swagger_route http_method, path, opts = {}
   add_swagger_route 'GET', '/workflows/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflow_by_id'
   add_swagger_route 'GET', '/workflows/{workflow_id}/requests', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflow_requests'
   add_swagger_route 'GET', '/workflows', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_workflows'
-  add_swagger_route 'DELETE', '/actions/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_action'
   add_swagger_route 'DELETE', '/groups/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_group'
-  add_swagger_route 'DELETE', '/requests/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_request'
-  add_swagger_route 'DELETE', '/stages/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_stage'
   add_swagger_route 'DELETE', '/templates/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_template'
   add_swagger_route 'DELETE', '/workflows/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'remove_workflow'
   add_swagger_route 'PATCH', '/actions/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'update_action'
```

Comment:
no patch for action. Only create is allowed.

Reply:
Removed